extensions/wikia/AssetsManager/config.php

```diff
@@ -215,6 +215,7 @@
 		'//extensions/wikia/AdEngine/js/context/adContext.js',
 		'//extensions/wikia/AdEngine/js/lookup/lookupFactory.js',
 		'//extensions/wikia/AdEngine/js/slot/adSlot.js',
+		'//extensions/wikia/AdEngine/js/slot/adUnitBuilder.js',
```

Comment:
But right now with `//extensions/wikia/ARecoveryEngine/js/helper.js` is the same situaton and we want to avoid situation when `helper.js` will be loaded without `adUnitBuilder.js`